packages/shared-business/src/components/InlineDatePicker.tsx

```diff
@@ -91,15 +99,21 @@ export const InlineDatePicker = ({
               year: date.year.trim(),
             }
           : undefined,
-      strategy: "onBlur",
+      strategy: "onSubmit",
```

Comment:
are you sure about that?

Reply:
Yes. Validation is triggered manually on fields actions (value change / blur). We can totally bypass strategies (`onSubmit` will never be fired, as `submitForm` is not used inside the component)